packages/EasyAsync/src/Bridge/Symfony/Resources/config/implementations/doctrine.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+use EonX\EasyAsync\Implementations\Doctrine\DBAL\DataCleaner;
+use EonX\EasyAsync\Implementations\Doctrine\DBAL\JobLogPersister;
+use EonX\EasyAsync\Implementations\Doctrine\DBAL\JobPersister;
+use EonX\EasyAsync\Interfaces\DataCleanerInterface;
+use EonX\EasyAsync\Interfaces\JobLogPersisterInterface;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+
+    $services->defaults()
+        ->autowire()
+        ->autoconfigure();
+
+    $services->set(DataCleanerInterface::class, DataCleaner::class);
+
+    $services->set(JobLogPersisterInterface::class, JobLogPersister::class)
+        ->bind('$table','%easy_async_job_logs_table%');
```

Comment:
Any particular reason why we're using `bind` instead of `arg`?

Reply:
Ok it was like that in the YAML file :D nvm

Reply:
Good question, `bind()` is confusing here for me too. I'm curious, what's the different in 
behavior?

I saw bind() only in defaults() definition above.

Reply:
End result is the same, at least for us 😄 Usually I use `arg` for a specific service but it might be one of those packages I've written overnight because of an idea I had, and I was young at the time! At least younger than now anyway... 😉